synIndex.R

```diff
@@ -2,19 +2,28 @@
 ## Index S3 Objects in Synapse
 ################################
 
+#############
+### Synapse credentials
+#############
+synapse_creds <- read.table('synapse_creds.txt', header = F, sep = '=', stringsAsFactors = F) 
+SYNAPSE_USERNAME <- synapse_creds$V2[1]
+SYNAPSE_PASSWORD <- synapse_creds$V2[2]
+
```

Comment:
See comment about storing credentials in code - Don't.

Reply:
Reading them in as environment variables set in .Renviron file.